odxtools/templates/macros/printDynamicLengthField.xml.jinja2

```diff
@@ -0,0 +1,21 @@
+{#- -*- mode: sgml; tab-width: 1; indent-tabs-mode: nil -*-
+ #
+ # SPDX-License-Identifier: MIT
+-#}
+
+{%- macro printDynamicLengthField(dlf) -%}
+<DYNAMIC-LENGTH-FIELD ID="{{dlf.odx_id.local_id}}">
+ <SHORT-NAME>{{dlf.short_name}}</SHORT-NAME>
+ <LONG-NAME>{{dlf.long_name|e}}</LONG-NAME>
```

Comment:
long_name is optional, description is missing

Reply:
right. fixed: 4eb810141661848e23f1e727ef9af9725e6892a0 (I took this as an opportunity and clean up ELEMENT-ID writing. The missing tags here were caused by copy-paste-and-modifying the END-OF-PDU-FIELD macro...)